pkg/core/native/crypto.go

```diff
@@ -273,6 +273,12 @@ func (c *Crypto) bls12381Equal(_ *interop.Context, args []stackitem.Item) stacki
 			panic(fmt.Errorf("equal: unexpected y bls12381 point type: %T vs G2Jac", y))
 		}
 		res = x.Equal(y)
+	case *bls12381.GT:
```

Comment:
You probably can reuse `BlsPoint.Equals` for `bls12381Equal` (and if you're to add `Bytes()` into it, then even reuse that for serialization.

Reply:
Fixed!